onebusaway-sdk-java-example/src/main/java/org/example/ArrivalAndDepartureForStop.java

```diff
@@ -0,0 +1,39 @@
+package org.example;
```

Comment:
should all of these packages be `org.onebusaway.example`?

Reply:
Yes, I have updated all the package names now.